client-side/src/axios/middleware.js

```diff
@@ -1,6 +1,7 @@
 import axios from 'axios';
 
-const url = process.env.REACT_APP_SERVER_URL;
+const url = process.env.REACT_APP_BASE_URL;

```

Comment:
why do you need the client url for axios?

Reply:
changed